src/main/java/org/javacord/bot/commands/MavenCommand.java

```diff
@@ -29,24 +39,25 @@ public void onCommand(Server server, TextChannel channel, Message message) {
             return;
         }
 
-        EmbedBuilder embed = new EmbedBuilder()
-                .setColor(Constants.JAVACORD_ORANGE)
-                .addField("Dependency",
-                        "```xml\n"
-                                + "<dependency>\n"
-                                + "    <groupId>org.javacord</groupId>\n"
-                                + "    <artifactId>javacord</artifactId>\n"
-                                // TODO Always use the latest version
-                                + "    <version>" + Javacord.VERSION + "</version>\n"
-                                + "    <type>pom</type>\n"
-                                + "</dependency>\n"
-                                + "```")
-                .addField("Setup Guides",
-                        "• [IntelliJ](https://javacord.org/wiki/getting-started/intellij-maven/)\n"
-                                + "• [Eclipse](https://javacord.org/wiki/getting-started/eclipse-maven/)");
-
-        CommandCleanupListener.insertResponseTracker(embed, message.getId());
-        channel.sendMessage(embed).join();
+        versionFinder.findLatestVersion().thenCompose(latestVersion -> {
```

Comment:
`join()` should be fine here. There's no real benefit of `thenComopose(...).join()` than just using `join()` directly.